Src/Extern/Conduit/AMReX_Conduit_Blueprint_ParticlesI.H

```diff
@@ -166,9 +166,10 @@ ParticleTileToBlueprint(const ParticleTile<NStructReal,
     // since these are contiguous arrays
 
     // array real fields
+    vname_real_idx = 0;
     for (int i = 0; i < NArrayReal; i++)
     {
-        conduit::Node &n_f = n_fields[real_comp_names[vname_real_idx]];
+        conduit::Node &n_f = n_fields[real_comp_names.at(vname_real_idx)];
```

Comment:
Not sure if `conduit::Node` supports something like `::insert()` to catch double-assignments?